docs/concepts/projects.md

```diff
@@ -61,42 +61,219 @@ Values that you place within your `deployment.yaml` file can reference dynamic v
+            name: my-docker-work-pool
+            work_queue_name: default
+    schedules:
+        every_ten_minutes: &every_10_minutes
+            interval: 600
+    actions:
+        docker_build: &docker_build
+            - prefect_docker.projects.steps.build_docker_image:
+                requires: prefect-docker>=0.2.0
+                image_name: my-example-image
+                tag: dev
+                dockerfile: auto
+                push: false
+
+deployments:
+  - name: deployment-1
+    entrypoint: flows/hello.py:my_flow
+    schedule: *every_10_minutes
+    parameters: 
+        number: 42,
+        message: Don't panic!
+    work_pool: *my-docker-work-pool
+    build: *docker_build
+
+  - name: deployment-2
+    entrypoint: flows/goodbye.py:my_other_flow
+    work_pool: *my-docker-work-pool
+    build: *docker_build
+
+  - name: deployment-3
+    entrypoint: flows/hello.py:yet_another_flow
+    schedule: *every_10_minutes
+    work_pool: 
+        name: my-process-work-pool
+        work_queue_name: primary-queue
+
+```
+
+In the above example, we are using YAML aliases to reuse work pool, schedule, and build configuration across multiple deployments:
+
+- deployment-1 and deployment-2 are using the same work pool configuration
+- deployment-1 and deployment-3 are using the same schedule 
+- deployment-1 and deployment-2 are using the same build deployment action
+
+### Deployment Declaration Reference
+
+#### Root Level Fields
+
+Below are fields that can be added at the root level of the `deployment.yaml` file.
+
+| Attribute | Description |
+| ---- | ---- |
+| `deployments` | A list of deployment declarations for the current project. Fields for this sections are documented in the [Deployment Fields](#deployment-fields) section. |
+| `definitions` | Definitions for configuration that is shared across deployment declarations (e.g., schedules, deployment actions, etc.). |
+
+#### Deployment Fields
+
+Below are fields that can be added to each deployment declaration.
+
+| Attribute | Description |
+| ---- | ---- |
+| `name` | A list of deployment declarations for the current project. |
+| `version` | An optional version for the deployment. |
+| `tags`    | A list of strings to assign to the deployment as tags. |
+| `description` | An optional description for the deployment |
+| `schedule`    | An optional [schedule](/concepts/schedule) to assign to the deployment. Fields for this sections are documented in the [Schedule Fields](#schedule-fields) section. |
+| `flow_name`   | The name of a flow that has been registered in the current project's [`.prefect` directory](#the-prefect-directory). Either `flow_name` **or** `entrypoint` is required. |
+| `entrypoint`  | The path relative to the project root to the file containing the flow to be deployed combined with the name of the flow function. Should be in the format `path/to.file:flow_function_name`. Either `flow_name` **or** `entrypoint` is required. |
```

Comment:
It might just be me but I found the `entrypoint` definition confusing. Maybe something more like:

"The file path of the flow you want to deploy, starting from the root directory of your project. Combined with the name of the flow function."